DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/CMakeLists.txt

```diff
@@ -170,8 +172,10 @@ elseif(DEVICE_FLAG MATCHES S10)
     set(FAMILY "Stratix 10")
 elseif(DEVICE_FLAG MATCHES CycloneV)
     set(FAMILY "Cyclone V")
+elseif(DEVICE_FLAG MATCHES Agilex5)
+    set(FAMILY "Agilex 5")
 else()
-    set(FAMILY "Agilex")
+    set(FAMILY "Agilex7")
```

Comment:
I think that this is incorrect, based on the fact that Agilex 5 and 7 are not written in the same way.
For this sample, I think that this should have been `Agilex` for when matching both `Agilex5` and `Agilex7`

Reply:
I'll open up an additional PR to address this issue.